boards/xtensa/intel_adsp_cavs18/doc/generic_intel_adsp.rst

```diff
@@ -61,6 +61,17 @@ also specify the rimage tool, config and key by:
               --config-dir [path to dir of .toml config file] \
               --key [path to signing key]
 
+
+Note: Sometimes you might have multiple network interfaces. In this case,
```

Comment:
```suggestion
Note: the current server script has a design issue where it listens only to a random network interface. This can be a problem if you have multiple interfaces or want to use localhost, in this case....
```